### DISCUSSION TITLE
Can you add example for NextJS Auth?

### DISCUSSION THREAD

**Discussion by michaldev** [fief-dev/fief#96] [Q&A]
NextJS is reactjs distribution with SSR. I would like to use Fief Auth with it.

**Comment by fief-bailiff** [comment]
Hail, @michaldev 👋 Welcome to Fief's kingdom!

Our team will get back to you very soon to help.

In the meantime, take a minute to star our repository ⭐️

![star-fief](https://user-images.githubusercontent.com/1144727/192523552-dc2c31fa-ae94-42fa-9ab4-7d0ba51bb7b0.png)

Farewell!

**Comment by frankie567** [comment]
Hello @michaldev 👋 

Right now, our Javascript SDK is not yet ready for NextJS or Express servers. This is something we have in our roadmap.

What you can try meanwhile is to follow the [React example](https://docs.fief.dev/integrate/javascript/react/). It should give you most of the authentication features, with some limitations due to NextJS.

I'll keep you tuned about our progresses about this 👍

**Comment by frankie567** [comment]
Hi @michaldev :wave:

We've released an official integration for Next.js. Here is the tutorial and example: https://docs.fief.dev/integrate/javascript/ssr/nextjs/

I'll happily hear your feedback about it!

Best!
